<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choice</class>
 <widget class="QDialog" name="choice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>254</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="enter_Button">
      <property name="text">
       <string>进入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="set_Button">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit_Button">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>179</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="room_label">
        <property name="text">
         <string>房间名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="room_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_line"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>填写信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
